docs/install/index.md

```diff
@@ -10,251 +10,22 @@ html_meta:
 
 # Install Plone 6
 
-This section focuses on installing Plone 6 for the developer, tester, or contributor.
+## Getting started
```

Comment:
Does any of the deleted content have value? If so, let's move it into the appropriate section.

Reply:
@stevepiercy I'm not 100% sure. The part about the Image installation method is definitely superseded by the containers subchapter. 

@fredvd I think we talked about this on the Beethoven sprint but I'm not sure if we came to a final conclusion regarding the existing installation docs.

Reply:
We did not reach an agreement. I agree about the containers chapters.

I think that the content has value. I moved it around and tweaked it a little bit in commit 7ef920155b3d09ae4f5355c3d61e1941be8e3313. I also marked it with a `TODO` for future consideration.

For the purpose of this PR—establishing the structure—I think we have gone about as far as we can until we have another huddle to discuss next steps. I am not happy with the structure of `source` in its current state, but maybe with content it will come together.

In any case, I am OK to merge this PR with my changes pushed in that commit.